data.rb

```diff
@@ -74,5 +81,18 @@ def data
     {:id=>49, :name=>"Rowland Wiegand", :age=>90, :apartment_id=>11},
     {:id=>50, :name=>"Rigoberto Armstrong IV", :age=>60, :apartment_id=>1}
   ]
+  class Apartments
+    attr_accessor :address, :monthly_rent, :square_feet
+  end
+
+  user = Teanant.new
+  @name = name
+  @age = age
+  @apartment_id = apartment_id
```

Comment:
All this stuff needs to go inside the class definition. But you closed it on line 86.